api/proxy_gen.go

```diff
@@ -454,6 +458,17 @@ func (s *BoostStub) BoostIndexerAnnounceAllDeals(p0 context.Context) error {
 	return ErrNotSupported
 }
 
+func (s *BoostStruct) BoostIndexerAnnounceDealRemoved(p0 context.Context, p1 cid.Cid) (cid.Cid, error) {
+	if s.Internal.BoostIndexerAnnounceDealRemoved == nil {
+		return *new(cid.Cid), ErrNotSupported
```

Comment:
I think you can just use `cid.Undef` instead of `*new(cid.Cid)`

Reply:
This is auto generated by the `make gen`. We might have to adjust the types in api gen for this. I am not sure if it is worth the required changes.

Reply:
Yes, sorry, didn't notice this is the auto-generated part of the API.